activerecord/test/fixtures/books.yml

```
@@ -11,6 +11,7 @@ awdr:
   font_size: :medium
   difficulty: :medium
   boolean_status: :enabled
+  cover: 'soft'
```

I don't know why Rubocop missed this but can you use double quotes?

```suggestion
  cover: "soft"
```